dist/file-preview-card/file-preview-card.css

```diff
@@ -0,0 +1,161 @@
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    height: 100%;
+    max-width: 396px;
+    min-width: 120px;
+    width: 100%;
+}
+
+.file-preview-card__body {
+    align-items: center;
+    display: flex;
+    grid-area: 1/1/4/4;
+    justify-content: center;
+}
+
+.file-preview-card__body img,
+.file-preview-card__body video {
+    border-radius: var(
+        --file-preview-card-border-radius,
+        var(--border-radius-100)
+    );
+    height: 100%;
+    object-fit: contain;
+    width: 100%;
+}
+
+.file-preview-card__body--see-more {
+    opacity: 0.5;
+}
+
+.file-preview-card__general-icon.icon {
+    height: 40%;
+    width: 40%;
+}
+
+.file-preview-card .file-preview-card__menu-action {
+    grid-area: menu-action;
+    justify-self: end;
+    z-index: 1;
+}
+
+.file-preview-card .file-preview-card__menu-action .icon-btn {
+    height: var(--spacing-400);
+    margin: var(--spacing-100);
+    width: var(--spacing-400);
+}
+
+.file-preview-card .file-preview-card__menu-action .icon-btn:not(:hover) {
+    background-color: var(
+        --file-preview-card-menu-action-background-color,
+        var(--color-background-primary)
+    );
+}
+
+.file-preview-card__preview-action {
+    align-self: end;
+    grid-area: label;
+    z-index: 1;
+}
+
+.file-preview-card__preview-action .icon-btn {
+    align-items: center;
+    background-color: var(
+        --file-preview-card-label-background-color,
+        var(--color-background-primary)
+    );
+    display: flex;
+    flex-direction: row;
+    gap: var(--spacing-50);
+    height: var(--spacing-250);
+    margin: var(--spacing-100);
+    padding: var(--spacing-25) var(--spacing-100);
+    width: max-content;
+}
+
+.file-preview-card__preview-action-text {
+    color: var(
+        --file-preview-card-label-text-color,
+        var(--color-foreground-primary)
+    );
+    font-size: var(--font-size-12);
+    font-weight: 400;
+}
+
+.file-preview-card__video-icon.icon {
+    height: var(--spacing-150);
+    width: var(--spacing-150);
+}
+
+.file-preview-card .file-preview-card__body-action {
+    align-self: center;
+    grid-area: center-action;
+    justify-self: center;
+    z-index: 1;
+}
+
+.file-preview-card .file-preview-card__body-action .icon-btn {
+    aspect-ratio: 1/1;
+    background-color: rgba(
+        var(
+            --file-preview-card-center-action-background-color-rgb,
+            var(--color-background-strong-rgb)
+        ),
+        0.5
+    );
+    color: var(
+        --file-preview-card-center-action-color,
+        var(--color-foreground-on-strong)
+    );
+    font-size: var(--font-size-14);
+    font-weight: 700;
+    height: -moz-fit-content;
+    height: fit-content;
+    line-height: var(--spacing-250);
+    padding: var(--spacing-100);
+    width: -moz-fit-content;
+    width: fit-content;
+}
+
+.file-preview-card .file-preview-card__body-action .icon-btn:active,
```

Comment:
Do we always need the BEM block class in the selector, in addition to the BEM element class? Curious if all this added specificity is required, or whether the cascade order will suffice. 

Can we replace this:

`.file-preview-card .file-preview-card__body-action .icon-btn:active`

With just this:

`.file-preview-card__body-action .icon-btn:active`